js/apiv2.js

```diff
@@ -91,8 +91,27 @@ const api = {
         return this.apiInstance.get(`/iptt/indicator/${indicatorPk}/`)
                     .then(response => response.data)
                     .catch(this.logFailure);
+    },
+    async checkSessions (query) {
+        return await axios.get('/update_user_session/', {
+            params: {
+                query: query
+            }
+        })
+        .then(response => response.data)
+        .catch(this.logFailure)
+    },
+    updateSessions (sessionVarsToUpdate) {
+        axios.put('/update_user_session/', sessionVarsToUpdate, {
```

Comment:
Would it make sense to put the axios call into a form similar to ones at the top of the file and then call it the way the ones above are called?